bin/fuel-core/src/cli/run.rs

```diff
@@ -165,10 +165,22 @@ pub struct Command {
     #[arg(long = "native-executor-version", env)]
     pub native_executor_version: Option<StateTransitionBytecodeVersion>,
 
+    /// The starting gas price for the network
+    #[arg(long = "starting-gas-price", default_value = "1000", env)]
+    pub starting_gas_price: u64,
```

Comment:
To be backward compatible the default starting price also should be zero and `gas-price-change-percent` should be zero as well.